server/sonar-ce/src/main/java/org/sonar/ce/app/StartupBarrier.java

```diff
@@ -20,7 +20,7 @@
 package org.sonar.ce.app;
 
 @FunctionalInterface
-public interface WebServerWatcher {
+interface StartupBarrier {
   /**
    * This blocking call, waits for the Web Server to be operational until either the Web Server is actually
```

Comment:
"[...] for the Web Server to be operational [...]" <- javadoc should be fixed since the interface is now generic